export.go

```diff
@@ -33,7 +39,15 @@ type Exporter struct {
 }
 
 // NewExporter creates a new Exporter. Callers must call Close() when done.
-func newExporter(tree *ImmutableTree) *Exporter {
+func newExporter(tree *ImmutableTree) (*Exporter, error) {
+	if tree == nil {
+		return nil, ErrorTreeNil
+	}
+	// CV Prevent crash on incrVersionReaders if tree.ndb == nil
+	if tree.ndb == nil {
+		return nil, ErrorTreeNodeDbNil
```

Comment:
maybe we can have one error type: `ErrNotIniitalizedTree`. And then use wrapping to add more details, eg:
```
ErrNotIniitalizedTree.Wrap("ndb is nil")
```